src/hooks/use-aggrid-row-filter.ts

```diff
@@ -119,11 +126,17 @@ export const useAggridRowFilter = (
 
             const { value, type, dataType } = selectedValue;
 
+            let backendValue = value;
+
+            if (dataType === 'number' && kiloValues.includes(field)) {
```

Comment:
We are using short-circuit specific data to update the value here, in a **common** component: this is not very good.
Could we try to use a callback 'updateValueCallbacl', passed only by short-circuit-analysis-result, to update the value ?